<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimUT</class>
 <widget class="QMainWindow" name="AnimUT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnimUT Framework</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/icons/images/icon.gif</normaloff>:/icons/images/icon.gif</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_controls">
      <property name="text">
       <string>Controls</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_visualization">
      <property name="text">
       <string>Visualization</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="Viewer" name="MainGraphicsView"/>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="ControlsLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAnimation">
    <property name="title">
     <string>Animation</string>
    </property>
    <addaction name="actionStartStop"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnimation"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionStartStop">
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About this project</string>
   </property>
   <property name="toolTip">
    <string>About AnimUT Framework</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Viewer</class>
   <extends>QGraphicsView</extends>
   <header location="global">AnimUT/Viewer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
